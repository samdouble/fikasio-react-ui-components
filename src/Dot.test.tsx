import React from 'react';
import { render } from '@testing-library/react';
import { Dot } from '.';

describe('Dot', () => {
  it('Renders correctly', () => {
    const { baseElement } = render(<Dot size={30} />);
    expect(baseElement).toMatchSnapshot();
  });
});
