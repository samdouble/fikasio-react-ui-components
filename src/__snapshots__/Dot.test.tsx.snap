// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dot Renders correctly 1`] = `
<span
  className="fikasio-dot fikasio-theme-dark"
  style={
    Object {
      "backgroundColor": "#000000",
      "height": 30,
      "width": 30,
    }
  }
/>
`;
